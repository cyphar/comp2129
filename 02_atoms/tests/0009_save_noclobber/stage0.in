START 5 10 10
SAVE save.as
