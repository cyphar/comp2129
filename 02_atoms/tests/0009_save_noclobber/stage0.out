Game Ready
Red's Turn

Game Saved

